### PR TITLE
CA certificate for TLS support

### PR DESCRIPTION
**Pull request by murat1985**
I suggest to add CA certificate support to this cookbook. We are using own CA certificate and probably a lot of people doing the same. 

Also I did slight style fixes and added libcap-bin package as it is not coming with ubuntu-12.04

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -26,6 +26,8 @@
 end
 
 vault_binary = File.join(service.install_path, 'vault', 'current', 'vault')
+
+package 'libcap2-bin'
```
Comment (johnbellone) [inline recipes/default.rb]: I'm not sure on what platform this package is required? Just based on the library name I assume the resource below this is affected and we probably want to hoist the guards for windows/mac to encompass that.

**Comment by johnbellone** [comment]
@murat1985 Please rebase and make changes I mentioned. After that this is :+1: to merge!

**Comment by murat1985** [comment]
Rebased with a guard for debian family platforms. CentOS 7 has this binary in distribution. Local kitchen tests are green, thanks.

**Comment by johnbellone** [comment]
Not sure but we're still getting the error... try rebasing again?

**Comment by johnbellone** [comment]
I am removing the deploying of certificates from this cookbook. It has caused a lot of problems, and over the past few months I have decided its better left for a wrapper cookbook. We will supply a wrapper cookbook called [vault-cluster](https://github.com/johnbellone/vault-cluster) which will deploy certificates using a custom resource.

**Pull request closed without merging**